api/errcode.proto

```diff
@@ -44,8 +44,9 @@ enum ErrCode {
   ErrCryptoSignature = 203;
   ErrCryptoSignatureVerification = 204;
   ErrCryptoDecrypt = 205;
-  ErrCryptoEncrypt = 206;
-  ErrCryptoKeyConversion = 207;
+  ErrCryptoDecryptPayload = 206;
```

Comment:
Why adding `ErrCryptoDecryptPayload` when `ErrCryptoDecrypt` is already available?

Reply:
It's to differentiate why the message can't be opened (`openEnvelopeHeader` ≠ `openPayload` in `keystore_message.go`) before adding it in the cache (I want add it only if `openPayload` failed).